### ISSUE TITLE
setting log level from config file

### ISSUE THREAD

**Issue by Dominik-Vogel** [qdev-dk/qdev-wrappers#40]
As far as I can see, the log level that is set int the qcodesrc is not used from within qcodes. Instead of setting the loglevel in the wrappers, here:
https://github.com/qdev-dk/qdev-wrappers/blob/5d08d07743a42ce2435a80158d01bbdc6f11fd92/qdev_wrappers/templates/initialisation_script.py#L45
we could set it in the init function and use the value from the config.

**Comment by jenshnielsen** [comment]
It would be great if all init scripts would execute `init_python_logger` that one should probably read the stream level from the config file. The fh1 one should be changed to rotate at `midnight` because the counter of daily

Init scripts should start with:
```
init_python_logger()
atexit.register(qc.Instrument.close_all())

```

**Comment by ThorvaldLarsen** [comment]
@Dominik-Vogel I assume this has been solved with #41?

**Comment by ThorvaldLarsen** [comment]
Pretty sure this was fixed long ago.
